test/langtools/tools/lib/toolbox/AbstractTask.java

```diff
@@ -83,17 +86,56 @@ public Result run(Expect expect, int exitCode) {
         return run();
     }
 
+    /**
+     * Sets the expected outcome of the task and calls {@code run()}.
+     * @param expect the expected outcome
+     * @param exitCodeValidator an exit code validator. The first parameter will
+     *                          be the actual exit code, the second test name,
+     *                          should throw TaskError is the exit code is not
```

Comment:
typo: "is the exit code" -> "if the exit code"

Reply:
Thanks, fixed now!